health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/LatLongMap/LatLongMap.js

```
@@ -0,0 +1,412 @@
+
+  const handleGeographyClick = (name, level, hasCoordinatesDown) => {
+    if (tableData.hasOwnProperty(name)) {setPointName(name); return};
+    if(drillDownChart !== "none") {
+      if (!hasCoordinatesDown) {
+        if(level===4){
+          setFilterFeature({finalFilter: name})
+        } else return;
+      }
+  
+      if (level === 2) {
+        let dummy = {...filterStack};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'dummy' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)